src/api/queryKey.ts

```diff
@@ -10,6 +10,15 @@ const userQueryKeys = {
   }),
 };
 
+const gameQueryKeys = {
+  tournamentHistory: (type: 'ROUND_2' | 'ROUND_4') => ({
+    _def: 'tournament-history',
+    queryKey: ['tournament-history', type],
+    queryFn: () => fetcher.get<HttpResponse<TournamentGameList>>('/game/history/${type}'),
```

Comment:
```suggestion
    queryFn: () => fetcher.get<HttpResponse<TournamentGameList>>('game/history/${type}'),
```

`ky` 를 사용하기 때문에 시작을 `/`로 하게 되면 네트워크 오류가 발생하는 것 같아요.